src/components/Avatar.js

```diff
@@ -81,6 +94,7 @@ class Avatar extends PureComponent {
                     )
                     : (
                         <Image
+                            key={this.props.source}
```

Comment:
What is the reason for this?

Reply:
There's a bug with react native that's the image won't reload itself one it load failed, to make it reload we need to pass a key to it.

Reply:
Sure.
FYI: https://github.com/facebook/react-native/issues/12606, https://github.com/facebook/react-native/issues/9195, https://medium.com/@wcandillon/5-things-to-know-about-images-react-native-69be41d2a9ee

Reply:
Can you please show me a video of this issue and explain me?

Reply:
@hungvu193 / @parasharrajat if you think we need / don't need it would you be able to test with / without it to prove so?

Let's keep this PR moving.

Reply:
@parasharrajat But sometimes I can't reproduce.

https://user-images.githubusercontent.com/16502320/206469911-17cdaa6f-8bff-4ac9-8f07-249b3b73f6b7.mov

Reply:
So what is the issue on this video? I didn't get it. Images were loaded after you went online which it should. :confused:

Reply:
Yeah, I mean sometimes I can't reproduce the `Image can't refresh without key`, I'll remove it for now